Services/Services/InterviewService.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Common.Listing;
+using Data.Entities;
+using Data.Repositories;
+using PagedList;
+using Services.DTOs.Interview;
+
+namespace Services.Services
+{
+    public class InterviewService
+    {
+        private InterviewRepository _interviewRepository;
+        private UserRepository _userRepository;
+        private CandidateRepository _candidateRepository;
+
+        public InterviewService(InterviewRepository interviewRepository, UserRepository userRepository, CandidateRepository candidateRepository)
+        {
+            _interviewRepository = interviewRepository;
+            _userRepository = userRepository;
+            _candidateRepository = candidateRepository;
+        }
+
+        public InterviewDTO Get(int interviewId)
+        {
+            Interview interview = _interviewRepository.GetById(interviewId);
+            if (interview == null)
+            {
+                return null;
+            }
+
+            InterviewDTO interviewDTO =
+                new InterviewDTO(
+                    interview.Id,
+                    interview.Date,
+                    interview.CandidateId,
+                    _candidateRepository.GetById(interview.CandidateId).Email,
```

Comment:
Create var for this email. Also create method to get only email from DB. Or you can add Name and Surname to DTO and get all 3 values from Candidate object